### DISCUSSION TITLE
Allowing editing of comment, issues, pull requests, without adding to edit history.

### DISCUSSION THREAD

**Discussion by yanickrochon** [community/community#49694] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Sorry if this is mislabelled, this is my first time commenting in here.

Often, I will write a comment on Github and, even after re-reading myself, I post and realise after that I have a typo. I "edit" my comment and this creates an edit history.

**Wouldn't it be nice to edit comments without having it marked as "edited"?**

An edit history would be added if :

1. some new activity appear after the comment
2. the comment has been inactive for ~5 minutes

**Comment by kasperk81** [comment]
i HATE this "feature"
> the comment has been inactive for ~5 minutes

THIS is how it should be! and this is how it works on stackoverflow, reddit and other places. have a cooldown time of 5-10 minutes of last edit before recording the history (which nobody should need in the first place).  the point of comment edit functionality is to correct a mistake. putting "penalty" on someone rephrasing a sentence or code, or fixing a typo is just mean and low. it has only negative effects on the ecosystem and there is no upside to it.
